src/redux/thunk.js

```diff
@@ -0,0 +1,18 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'http://localhost:3000/messages/random';
```

Comment:
- [ ] Hi @unorjikingsley, as mentioned by the previous reviewer, I think your two apps are not successfully connecting together. The front-end app is not consuming the API as you'll see the comments in the backend PR.
- [Optional] Please it is a good practice to versioned the API. This enable user to keep using the existing REST API while only updating applications when needs arises. 
- Please kindly check the other comments at the backend :+1: